src/components/__tests__/Pagination-test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Pagination from '../Pagination'
+
+describe('components/Pagination', () => {
+  const totalPage = 5
+  const totalNumButton = 3
+  const activePage = 2
+  const onSelect = jest.fn()
+
+  it('renders successfully', () => {
+    const pagination = shallow(
+      <Pagination
+        totalPage={totalPage}
+        totalNumButton={totalNumButton}
+        activePage={activePage}
+        onSelect={onSelect}
+      />
+    )
+
+    expect(pagination.containsMatchingElement(
+      <div>
+        <ul>
+          <li><a>« First</a></li>
+          <li><a>« Prev</a></li>
```

Comment:
I'm confused, shouldn't the previous be rendered with a less than sign like:

< Prev